common/lib/xmodule/xmodule/js/src/sequence/display.js

```diff
@@ -38,6 +38,12 @@
             this.displayTabTooltip = function(event) {
                 return Sequence.prototype.displayTabTooltip.apply(self, [event]);
             };
+            this.arrowKeys = {
```

Comment:
I would also like to see this, but I believe it isn't possible without requirejs because we don't expose `constants` on `window.edx`.